src/awkward/operations/ak_to_feather.py

```diff
@@ -0,0 +1,130 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+__all__ = ("to_feather",)
+from os import fsdecode
+
+import awkward as ak
+from awkward._dispatch import high_level_function
+from awkward._nplikes.numpylike import NumpyMetadata
+
+metadata = NumpyMetadata.instance()
+
+
+@high_level_function()
+def to_feather(
+    array,
+    destination,
+    *,
+    list_to32=False,
+    string_to32=True,
+    bytestring_to32=True,
+    emptyarray_to=None,
+    categorical_as_dictionary=False,
+    extensionarray=True,
+    count_nulls=True,
+    compression="zstd",
+    compression_level=None,
+    chunksize=None,
+    feather_version=2,
+):
+    """
+    Args:
+        array (pandas.DataFrame, or pyarrow.Table): Data to write out as Feather format,
+            passed to [pyarrow.feather.write_feather](https://arrow.apache.org/docs/python/generated/pyarrow.feather.write_feather.html#pyarrow.feather.write_feather).
+        destination (str): Local destination path, passed to [pyarrow.feather.write_feather](https://arrow.apache.org/docs/python/generated/pyarrow.feather.write_feather.html#pyarrow.feather.write_feather).
+        compression (str): Can be one of {“zstd”, “lz4”, “uncompressed”}. The
+            default of None uses LZ4 for V2 files if it is available, otherwise
+            uncompressed. Passed to [pyarrow.feather.write_feather](https://arrow.apache.org/docs/python/generated/pyarrow.feather.write_feather.html#pyarrow.feather.write_feather).
+        compression_level (int): Use a compression level particular to the chosen
+            compressor. If None use the default compression level. Passed to [pyarrow.feather.write_feather](https://arrow.apache.org/docs/python/generated/pyarrow.feather.write_feather.html#pyarrow.feather.write_feather).
+        chunksize (int): For V2 files, the internal maximum size of Arrow RecordBatch
+            chunks when writing the Arrow IPC file format. None means use the
+            default, which is currently 64K. Passed to [pyarrow.feather.write_feather](https://arrow.apache.org/docs/python/generated/pyarrow.feather.write_feather.html#pyarrow.feather.write_feather).
+        version (int): Feather file version, passed to [pyarrow.feather.write_feather](https://arrow.apache.org/docs/python/generated/pyarrow.feather.write_feather.html#pyarrow.feather.write_feather).
+            Version 2 is the current. Version 1 is the more limited legacy format. If not
+            provided, version 2 is used.
+
```

Comment:
Not all of the arguments have been documented in this docstring. They all have the same meaning in other functions, so you can find those other functions and copy-paste the docstring parts here. Also notice that the argument to this function is `feather_version`, not `version`. (These are all good examples of things that would confuse a reader of the documentation.)